arbitrum-docs/for-devs/dev-tools-and-resources/debugging-tools.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Debugging tools"
+sidebar_label: Debugging tools
+description: This page describes some of the popular debugging tools that exist in the Arbitrum ecosystem
+reader-audience: developers who want to build on Ethereum/Arbitrum
+content-type: overview
+---
+
+import KnowMoreToolsBox from '../partials/_know-more-tools-box-partial.md';
+
+<KnowMoreToolsBox />
+
+Developing and maintaining decentralized applications on any blockchain can be a complex task, as debugging transactions on-chain isn't always a user-friendly process. In this section we cover a set of tools that can greatly enhance the development and troubleshooting experience. Leveraging these solutions can help developers swiftly identify and resolve issues, ensuring robust and efficient decentralized applications for the ever-growing blockchain ecosystem.
+
+### Tenderly
+
+[Tenderly](https://tenderly.co/) is an all-in-one Web3 development platform that empowers developers to build, test, monitor, and operate smart contracts from inception to mass adoption. Tenderly's debugging options focus on providing developers with efficient and user-friendly tools to identify and fix smart contract bugs and production issues. The Debugger enables developers to inspect smart contracts by analyzing precise lines of code in a human-readable format. With Tenderly's Simulator, developers can play out specific historical transactions and current transaction outcomes before sending them on-chain, allowing them to change relevant parameters and source code to test and debug contracts. The platform streamlines the debugging process, saving time and resources while improving smart contract reliability.
```

Comment:
Should we talk about Tenderly being buggy when it comes to custom Geth errors and link to this maybe?
https://www.notion.so/arbitrum/I-tried-to-create-a-retryable-ticket-but-the-transaction-reverted-on-L1-How-can-I-debug-the-issue-606f4ef84f884f4cb940ffc483e7b2ec#c5ded8ccec6543309fdf4086c8ef4214

Reply:
These are meant to be general descriptions of those tools, but that gives me an idea. We can create a "How to debug Arbitrum transactions" page, where we describe the process and mention what you just said. Then, from Tenderly's description, we can link to that How to.
The How to is likely to get more traffic from devs anyway, and that FAQ will be more useful there, imo. What do you think?

Reply:
Added a note on Tenderly's limitation. Thanks!